app/models/cqm/measure.rb

```diff
@@ -37,5 +42,7 @@ class Measure
     field :main_cql_library, type: String
 
     embeds_many :source_data_criteria, class_name: 'CQM::DataElement'
+
+    embeds_many :population_sets
```

Comment:
It would be nice to cover the new fields and embedded entities with a unit test to make sure it can be persisted in mongo

Reply:
Makes sense. At the moment, I'm not sure how to parse the measure into the population sets. To avoid potential issues, I'll remove the reference for now, then add it back if its needed.